frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -20,19 +21,47 @@ export const detectIndependentMode: FeatureDetector = (dispatch) =>
     },
   );
 
-export const OCS_VERSION_4_5_FLAG = 'OCS_VERSION_4_5_FLAG';
+export const OCS_SUPPORT_FLAGS = {
+  SNAPSHOT: 'SNAPSHOT',
+  EXTERNAL: 'EXTERNAL',
```

Comment:
@bipuladh we should keep the Flag behaviour consistent with other plugins, which is using capital letters, to avoid confusion, I have added a comment https://github.com/openshift/console/pull/5079/files#diff-28169bef773f6ff352cf998e558e1e02R24

Reply:
@a2batic the flag name is something that is defined in the console. These values are coming from CSV. The keys are uppercase and the keys can be used for creating flag names as I have put an example in my previous comment.